packages/font-icons-mdl2/package.json

```diff
@@ -31,6 +31,12 @@
     "@fluentui/utilities": "^8.8.3",
     "tslib": "^2.1.0"
   },
+  "peerDependencies": {
+    "@types/react": ">=16.8.0 <18.0.0",
+    "@types/react-dom": ">=16.8.0 <18.0.0",
```

Comment:
@types/* are devDependencies and shouldn't be required to be installed by any downstream dependency. Right?

Reply:
It is feasible that font-icons-mdl2 could have react/react-dom as a `devDependency` if we were defining strict dependencies. This might be required for running tests at this layer in isolation, which would exercise the utilities paths. Peer dependencies would then be "resolved" at this point.

But a change like that should not affect downstream repos. A downstream repo who installs font-icons-mdl2 will still need to resolve the transient peer deps at that layer regardless of the resolution of the intermediate layers which don't directly depend on react.